kotest-assertions/kotest-assertions-core/api/kotest-assertions-core.api

```diff
@@ -473,11 +473,25 @@ public final class io/kotest/matchers/collections/EmptyKt {
 	public static final fun beEmpty ()Lio/kotest/matchers/Matcher;
 	public static final fun beEmptyArray ()Lio/kotest/matchers/Matcher;
 	public static final fun shouldBeEmpty (Ljava/lang/Iterable;)Ljava/lang/Iterable;
-	public static final fun shouldBeEmpty (Ljava/util/Collection;)Ljava/util/Collection;
```

Comment:
It seems a collection override has gone missing

Reply:
as Collections are Iterable I suppose we are fine?

Reply:
Yes, that is the intended effect of this PR. Having a Collection version in addition to Iterable version has the only effect of duplicating IDE autocompletion for library consumers.

Removal of the Collection override is a non-breaking change. Consumer code will simply start resolving to the Iterable version. Their import statement is identical - `io.kotest.matchers.collections.shouldBeEmpty`.

Before:
<img width="857" alt="image" src="https://github.com/user-attachments/assets/63d16c08-3d25-4720-a354-5dc655e549a8">

After:
<img width="857" alt="image" src="https://github.com/user-attachments/assets/cea2edd1-f9f9-4416-82db-98db8e3d01a8">